release_tester/arangodb/starter/deployments/dc2dc.py

```diff
@@ -385,8 +387,8 @@ def _mitigate_known_issues(self, last_sync_output):
                 )
             else:
                 self.progress(True, "arangosync: restarting instances...")
-                self.cluster1["instance"].kill_sync_processes()
-                self.cluster2["instance"].kill_sync_processes()
+                self.cluster1["instance"].kill_sync_processes(True, SYNC_VERSIONS["140"])
```

Comment:
you compare `SYNC_VERSIONS["140"]` to Starter version inside `kill_sync_process` ?

Reply:
yes, I probably should change this.